sklearn/metrics/scorer.py

```diff
@@ -290,7 +307,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     Examples
     --------
     >>> from sklearn.metrics import fbeta_score, make_scorer
-    >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
+    >>> ftwo_scorer = make_scorer(fbeta_score, name='fbeta_score', beta=2)
```

Comment:
I wasn't even thinking about the `fbeta_score(beta=2)` case. Thank you for bringing it up.

But at least from my perspective, given the current implementation with `name="custom_score"`, a user will 100% of the time specify the name of the metric if he/she wants to be specific. On the other hand, the implementation I suggested requires a user to specify the metric only _sometimes_.

That's just my two cents.